openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -1,6 +1,13 @@
 {% load tz %}
 {% load i18n %}
 {% spaceless %}
+{% load static %}
+
+<link rel="stylesheet" href="{% static 'css/codemirror.min.css'  %}">
+<script src="{% static 'js/lib/codemirror/codemirror.min.js' %}"></script>
+<script src="{% static 'js/lib/codemirror/clike.min.js' %}"></script>
+<script src="{% static 'js/lib/codemirror/python.min.js' %}"></script>
+
```

Comment:
I was not successful in including these files via require. So I had to include them as static files.